### ISSUE TITLE
NOW TV Apple TV

### ISSUE THREAD

**Issue by ClareDavidson** [ClareDavidson/accessibility#11]
However when I open the Now TV app, VoiceOver appears to be completely unable to interact with it.

The screen I see is in the attached photo, but VO seems to just see the whole screen as one element. It says something like “Hey You”. I cannot swipe to any elements on the screen, meaning the app is 100% un-usable to me currently.

 

VoiceOver support is something that has been raised with our development teams many times in the past.

I myself sent an email back in May when this trial was first promoted, which I have attached to this mail.

In addition I emailed Michael Forry from the Now TV ROI team over a year ago. Although he is from the marketing team, he did appear to take the feedback seriously.

 

Apple TV is an extremely accessible platform, with Apple providing the tools to make apps fully inclusive.

 

If this app goes to the public without being accessible, it would be to my mind completely unacceptable, and we are likely to receive some very harsh criticism from that user group.

**Comment by ClareDavidson** [comment]
Hi Marianne/David,

Apologies for the delay in my response.

I’ve spoken to the Scrum Master this morning, and he has advised that Voiceover isn’t something which was in scope for the NOW TV Germany app (which the new UK and Ireland apps are being based on).

This is why it doesn’t feature in the app at the moment.

That said, there is some positive news – thanks to David’s feedback, it is now something which the team are thinking about and there are ideas about how it might be fixed/implemented.

I’ll hopefully be speaking to the Product Owner this morning this morning to check on the requirement and priority – but it is now on the radar with a view to being implemented.

**Comment by ClareDavidson** [comment]
NOW TV have confirmed that they won't go live until VO fixed on the app. No further progress on sourcing an Apple TV.
